src/components/WalletAddress.js

```diff
@@ -71,7 +71,8 @@ export class WalletAddress extends React.Component {
    */
   generateNewAddress = async (e) => {
     e.preventDefault();
-    const address = await this.props.wallet.getNextAddress();
+    const wallet = getGlobalWallet();
+    const address = await wallet.getNextAddress();
```

Comment:
```suggestion
    const { getNextAddress } = getGlobalWallet();
    const address = await getNextAddress();
```

Reply:
It's not possible to use this syntax with a HathorWallet method: it crashes the application.